iidm/iidm-mergingview/src/test/java/com/powsybl/iidm/mergingview/TieLineAdapterTest.java

```diff
@@ -35,11 +35,11 @@ public void testSetterGetter() {
         assertNotNull(tieLine.getHalf1());
         assertNotNull(tieLine.getHalf2());
 
-        assertEquals(Branch.Side.ONE, tieLine.getHalf1().getBoundary().getSide());
+        assertSame(tieLine.getHalf1().getId(), tieLine.getHalf1().getBoundary().getDanglingLine().getId());
         assertSame(tieLine, tieLine.getHalf1().getBoundary().getConnectable());
         assertSame(tieLine.getTerminal1().getVoltageLevel(), tieLine.getHalf1().getBoundary().getNetworkSideVoltageLevel());
 
-        assertEquals(Branch.Side.TWO, tieLine.getHalf2().getBoundary().getSide());
+        assertSame(tieLine.getHalf2().getId(), tieLine.getHalf2().getBoundary().getDanglingLine().getId());
```

Comment:
You can assert for the objects themselves, not the ids

Reply:
Done